packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -284,12 +284,54 @@ class NestedScrollView extends StatefulWidget {
   }
 
   @override
-  _NestedScrollViewState createState() => _NestedScrollViewState();
+  NestedScrollViewState createState() => NestedScrollViewState();
 }
 
-class _NestedScrollViewState extends State<NestedScrollView> {
+/// This is the [State] for [NestedScrollView]. It allows you to obtain the scroll
```

Comment:
It really gives you access to the ScrollControllers, not directly the positions.